### ISSUE TITLE
pull access denied for lmb-freiburg-netdef

### ISSUE THREAD

**Issue by ZhangShuoAlreadyExists** [lmb-freiburg/netdef-docker#16]
Hi,

I want to run DispNet CSS model and I found this project is very helpful.
May I know if the docker container still exists? The script run-network.sh can't pull the container.

Unable to find image 'lmb-freiburg-netdef:latest' locally
docker: Error response from daemon: pull access denied for lmb-freiburg-netdef, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.

**Comment by nikolausmayer** [comment]
I do not believe the images were ever pushed to the Docker hub, so a pull command is not likely to succeed.
Did you perhaps just forget to `make`? :slightly_smiling_face:
